cPP_DSC.adoc

```diff
@@ -1023,6 +1023,94 @@ FCS_CKM.2.1:: The TSF shall distribute cryptographic keys in accordance with a s
++
+*_Key Access Operations_*
++
+_There are several reasons for granting access to keys or restricting access to keys.  The list below enumerates typical actions for keys in a typical lifecycle of a key.  Further on in this note are listed some methods used to authenticate roles and to authorize them to perform these actions._
++
+_The typical actions for the lifecycle of a key may look something like:_
++
+--
+. _Creation_
+. _Use in cryptographic operation_
+. _Archival, Backup and Escrow_
+. _Recovery_
+. _Attribute modification_
+. _Destruction_
+--
++
+*_Creation of cryptographic keys* - It may be the case that during the creation of the key certain attributes are assigned to it that controls future access to the key, such as who is allowed to use it or restricted from using it, when they are allowed to use it or restricted from using it, who is allowed to backup and recover keys, who is allowed to destroy them, and who is allowed to modify attributes associated with the keys.  This SFR imposes no requirements on the creation of cryptographic keys, other than those specified already in FCS_CKM.1.  However, once created, this SFR does impose requirements to access keys for the purposes outlined in the first assignment._
++
+*_Cryptographic key usage in cryptographic operations* - Depending on the application, a cryptographic key could be root key which by virtue of being a root key, cannot be encrypted with another key.  It is often stored in plain sight, sometimes with hardware protections, sometimes obfuscated in clever ways, such as in physically unclonable functions.  Other than root keys, it is expected that other cryptographic keys should be protected with encryption, often using other keys, which leads to a chain or hierarchy of cryptographic keys that encrypt other keys often anchored by a root key.   Additionally, it is expected that an entity authenticate itself self as an authorized user of the key.  This latter requirement is in addition to the recommended requirements of the Common Criteria Part 2 for FCS_CKM.3._
++ 
+*_Cryptographic key archival, backup, and escrow* - TOEs often perform cryptographic key archival to manage limited memory resources inside its own security boundaries.  Often the TOE encrypts the cryptographic keys prior to saving them to storage that is close by on the same device as the TOE and that can be accessed quickly. TOEs may perform cryptographic key backups into storage that is meant for longer term keeping.  Backup storage is often not on the same device as the TOE and may be physically hundreds of miles away.  In practice, the TOE encrypts the keys using an approved method before sending them to backup storage.  TOEs may perform cryptographic key escrow in which it entrusts a third party with access to the private or secret keys.  In practice, the TOE protects the keys using a cryptographic key access method agreed upon with the escrow agent before sending them._
```

Comment:
On escrow: Should the escrow mechanism be linked to some objective for the environment or OSP considering that the TOE has no means to enforce this?